docs/guides/migrating-from-camunda-7/data-migrator/runtime.md

```diff
@@ -223,18 +225,17 @@ This approach ensures that:
 ### Example
 
 Use the `camunda.migrator.tenant=ids` [property](/guides/migrating-from-camunda-7/data-migrator/config-properties.md#camundamigrator)
-to specify which tenants should be included in the
-migration process. This property accepts a comma-separated list of tenant identifiers and doesn't
-need to add the `<default>` tenant explicitly, as it is included out-of-the-box.
+to specify which tenants should be included in the migration process. This property accepts a
+comma-separated list of tenant identifiers.
 
 ```yaml
 camunda:
   migrator:
-    tenant-ids: tenant-1, tenant-2, tenant-3 # There's no need to specify `<default>` tenant id, as it included by default.
+    tenant-ids: tenant-1, tenant-2, tenant-3
 ```
 
-With this configuration, only process instances associated with `tenant-1`, `tenant-2`, `tenant-3`
-and `<default>` will be migrated. Instances associated with other tenants will be skipped.
+With this configuration, only process instances associated with `tenant-1`, `tenant-2`, `tenant-3`,
+and `<default>` will be created and migrated. Instances associated with other tenants will be skipped.
```

Comment:
❓ Couldn't this cause confusion?  Based on the wording I would expect that a process instance with tenant `<default>` will be migrated successfully, but that is not the case.
```suggestion
and the default tenant will be created and migrated. Instances associated with other tenants will be skipped.
```